src/components/VisitsPanelList/index.js

```diff
@@ -1,11 +1,34 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./styles.scss";
 import Router from "next/router";
 import formatDateAndTime from "../../helpers/formatDateAndTime";
 import VisitSummaryList from "../VisitSummaryList";
 import { GridRow, GridColumn } from "../Grid";
 import filterTodaysVisits from "../../helpers/filterTodaysVisits";
 import filterUpcomingVisits from "../../helpers/filterUpcomingVisits";
+import moment from "moment";
+
+const Beside = ({ callDateAndTime }) => {
```

Comment:
Should we extract this out to its own component file ?

Reply:
That's a good shout! I like dis. Me do dis.

Reply:
Created a `TimeFromNow` component which will return the time from now that will update every 1 minute by default but can be set.